src/commands/config.js

```diff
@@ -0,0 +1,43 @@
+const fs = require("fs");
+const os = require("os");
+const promptAsync = require("../util/promptAsync");
+
+const hiddenMaestroDirPath = () => {
+  const homedir = os.homedir();
+  const dir = "/.maestro";
+
+  return homedir + dir;
+};
+
+const createHiddenMaestroDir = () => {
+  const hiddenDirPath = hiddenMaestroDirPath();
+
+  if (!fs.existsSync(hiddenDirPath)) {
+    fs.mkdirSync(hiddenDirPath);
+  }
+};
+
+const asyncPromptForAccountInfo = async () => {
+  const accountNum = await promptAsync(
+    "Please enter your AWS Account Number: "
+  );
+  const region = await promptAsync(
+    "Please enter the region for you AWS servises (e.g. us-west-2): "
+  );
```

Comment:
I think it would be nice to show the values that are already in the file, assuming the file exists. So for example, the first prompt would probably look like this:

``` js
const { accountNum: existingAccountNum, region: existingRegion } = JSON.parse(fs.readFileSync(hiddenMaestroDirPath() + "aws_account_info.json"));
const accountNum = await promptAsync(
  "Please enter your AWS Account Number",
  existingAccountNum,
) || existingAccountNum;
```

This would work because the `promptAsync` takes additional arguments which it then displays as options, so the above would display this as a prompt (assuming "123456789012" was the `existingAccountNum`:

```
Please enter your AWS Account Number [123456789012] 
```

And then if the user just pressed enter, they would use the existing account number instead of empty string.

Reply:
Note that this is a suggestion\; just a nice optional feature.

Reply:
Hmm, I'm not sure if it's necessary especially assuming that the premise is that on the initial setup, the file didn't previously exist and that's the most likely time the command will be run.

Reply:
The behavior of the command is currently the same whether the file exists. Might be nice to display the existing info if the file is already there.

Reply:
Right, imagine the scenario where you need to update your region. With the current implementation, the user needs to retype their AWS account number (after tracking it down), which is a bother and the user may have a typo from something they don't need to update.